src/services/CatServices.ts

```diff
@@ -1,41 +1,37 @@
-import axios from "axios";
-import { CatList } from "../types";
+import axiosClient from "../utils/axiosClient";
+import { CatSummaryList } from "../types";
 import { deepCamelCaseKeys, deepSnakeCaseKeys } from "../utils/utils";
 
 const getAllCats = async (
-  pageNumber: Number,
-  pageSize: Number
-): Promise<any> => {
-  let url = "http://localhost:10000/v1/cats?sort_by=id";
-  url = url + "&page_number=" + pageNumber;
-  url = url + "&page_size=" + pageSize;
-
-  const { data } = await axios.get(url);
-
-  const result = deepCamelCaseKeys(data);
-  return result;
+  pageNumber: number,
+  pageSize: number,
+  sortBy: string
```

Comment:
[follow-up on comment above]
```suggestion
  {
    pageNumber,
    pageSize,
    sortBy = "id",
  } : {
    pageNumber: number,
    pageSize: number,
    sortBy?: "id" | "name", // This is called a "literal type" and useful if you want to restrict what kind of values can be passed (i.e. `sortBy: "numberOfWheels"` makes no sense)
  }
```
^ I agree that this is a bit more verbose but the order of arguments is no longer important and we provide means now to give a bit more context when we pass parameters.

Reply:
Thanks for this info. This is very helpful.